### PR TITLE
Rui integrate

### PR DESCRIPTION
**Pull request by etlds**

### CODE REVIEW

File: src/ingest-ui/src/components/uuid/tissue_form_components/ruiIntegration.jsx [hunk 1]
```diff
@@ -85,20 +60,21 @@ class RUIIntegration extends Component {
...
     window.ruiConfig = {
       // Custom configuration
-      embedded: false,
+      embedded: true,
       tutorialMode: false,
       homeUrl: '/donors-samples',
-      user: {
-        firstName: "Desheng",
-        lastName: "Li"
+      organ: {
```
Comment (bherr2) [inline src/ingest-ui/src/components/uuid/tissue_form_components/ruiIntegration.jsx:66]: Will you be able to pull this config from the sample/donor config?

**Comment by bherr2** [comment]
The staging site should be ready to test with the organ config. Let me know if it works.

File: src/ingest-ui/src/components/uuid/tissue_form_components/ruiIntegration.jsx [hunk 2]
```diff
@@ -36,31 +36,6 @@ class RUIIntegration extends Component {
...
-    //   self.handleCloseScreenClick();
-    //   //self.setState({close_rui:false});
-    // };
-    // var css_file = document.createElement("link");
-    // css_file.ref = "stylesheet";
-    // css_file.type = "text/css";
-    // css_file.href = "https://cdn.jsdelivr.net/gh/hubmapconsortium/ccf-ui@staging/rui/styles.css";
-    // document.head.appendChild(css_file);
     const runtime_script = document.createElement("script");
     runtime_script.src = "https://cdn.jsdelivr.net/gh/hubmapconsortium/ccf-ui@staging/rui/runtime.js";
```
Comment (bherr2) [inline src/ingest-ui/src/components/uuid/tissue_form_components/ruiIntegration.jsx:40]: You'll probably want to move the base url to a variable/config option (`https://cdn.jsdelivr.net/gh/hubmapconsortium/ccf-ui@staging/rui/` will likely change to `https://cdn.jsdelivr.net/gh/hubmapconsortium/ccf-ui@1/rui/` when we move to production.

File: src/ingest-ui/src/components/uuid/tissue_form_components/ruiIntegration.jsx (hunk 1 above)
Comment (etlds) [reply src/ingest-ui/src/components/uuid/tissue_form_components/ruiIntegration.jsx:66]: It is not straightforward. I have to read the suffix from the source Display ID and parse it to the organ config. E.g TEST0036-LK => orgain: { name: "kidney", side: "left", sex: "" }. There is no way to tell what sex is.

Comment (bherr2) [reply src/ingest-ui/src/components/uuid/tissue_form_components/ruiIntegration.jsx:66]: You can't get it from the donor block associated with the tissue? If not, please have a default (female) for when you can't do that.

Comment (etlds) [reply src/ingest-ui/src/components/uuid/tissue_form_components/ruiIntegration.jsx:66]: No, we don't fill in "sex" when registering a donor.

**Comment by etlds** [comment]
> The staging site should be ready to test with the organ config. Let me know if it works.

I still do not see the 3D organ and the "Review and Register" button will submit the tissue form. I found that if I not use embedded mode the "Review and Download" button will NOT submit the tissue form.

**Pull request merged**